nexus/src/app/sagas/instance_create.rs

```diff
@@ -373,131 +372,36 @@ async fn sic_add_network_config(
         &params.serialized_authn,
     );
     let osagactx = sagactx.user_data();
-    let dpd_client: &dpd_client::Client = &osagactx.nexus().dpd_client;
-    let datastore = &osagactx.datastore();
-    let log = sagactx.user_data().log();
-
-    let (.., authz_instance, db_instance) = LookupPath::new(&opctx, &datastore)
+    let datastore = osagactx.datastore();
+    let (.., db_instance) = LookupPath::new(&opctx, &datastore)
         .instance_id(instance_id)
         .fetch()
         .await
-        .map_err(ActionError::action_failed)?;
+        .map_err(ActionError::action_failed)
+        .unwrap();
```

Comment:
why not `?` ?

Reply:
Good catch, fixed in 024360f7. The instance record must exist at this point (it was created in the Creating state and can't move to a state that allows deletion until the saga finishes), but that doesn't mean the saga should assume the fetch will succeed.